src/controllers/Requests.js

```diff
@@ -0,0 +1,33 @@
+import db from '../models/index';
+import ValidateParams from '../validation/ValidateParams';
+/**
+ * class Requests defines travel requests endpoints
+ * @class
+ */
+class Requests {
+  /**
+   * @param {Oject} req is the request object
+   * @param {Object} res is the response object
+   * @static
+   * @returns {void}
+   */
+  static async getAUserRequest(req, res) {
+    const { userId } = req.params;
+
+    try {
+      ValidateParams.evaluate(userId);
```

Comment:
pls move validations to middleware

Reply:
the validations are in the appropriate folder designated for it validations. It is not a middleware